### ISSUE TITLE
Notifications not shown

### ISSUE THREAD

**Issue by pelennart** [humhub/humhub#1326]
![bildschirmfoto 2015-11-24 um 13 41 17](https://cloud.githubusercontent.com/assets/14274679/11367298/fcfc3cca-92b2-11e5-949e-3fcb5a687062.png)

For some users I only get the load splashscreen for system messages. Only some users are effected and sometimes the problem vanishes for unknown reasons and might show up for others at some point (I haven't figured). It's not system dependant, as I can login to other users on the same system and it works.

I'm running HumHub 0.20.1 with no custom modules or themes. I tried rebuilding the search index but it doesn't help.
The main Problem is, users can't accept space invites.

**Comment by pelennart** [comment]
Maybe the screenshot is not the best. If you look closely there's a tiny blue dot in the white space showing that there is indeed the load splashscreen animation. I promise!

**Comment by luke-** [comment]
Did you upgraded to 0.20.1?

Is there any error message:
- Browser Console
- Humhub Administration -> Logging
- WebServer/PHP Error Logs

**Comment by pelennart** [comment]
I opened console and this is what I got:
```
GET https://domainname/index.php?r=notification%2Flist%2Findex&from=0&ajax=1
```

Humhub logging is somewhat more detailled:
```
$_GET = [ 'r' => 'notification/list/index' 'from' => '0' 'ajax' => '1' ] $_COOKIE = [ 'language' => '32e2115501b4a47845f68a6a07b73d99137bb6b8722808215f5a98e7f6acd5b7a:2:{i:0;s:8:\"language\";i:1;s:2:\"de\";}' 'mp_b5e6ddf58b2d02245a7a19005d1cec48_mixpanel' => '{\"distinct_id\": \"150a9fc0508119-055c9ee5a67315-48516a-1fa400-150a9fc0509271\",\"$initial_referrer\": \"https://www.domainname.de:8000/\",\"$initial_referring_domain\": \"www.domainname.de:8000\"}' 'PHPSESSID' => 't72gum1ifnceui6akpbmbtae73' '_csrf' => '2afd7ac664c5802e76463c4af56452b8f9b40e720494d18287866ee4d2435c74a:2:{i:0;s:5:\"_csrf\";i:1;s:32:\"nXcw01HFuLyJHj4lffrQtHgxA1oE2W1H\";}' 'pm_groups-statistics-panel' => 'expanded' 'pm_new-spaces-panel' => 'expanded' 'pm_getting-started-panel' => 'expanded' 'pm_space-members-panel' => 'expanded' 'pm_user-spaces-panel' => 'expanded' 'pm_profile-following-panel' => 'expanded' '_identity' => 'ef55eb012b22a134ac156aecbcf0b9e68f3a062d503cb7a206ac5ddc5f0d2bbda:2:{i:0;s:9:\"_identity\";i:1;s:50:\"[2,\"00cf18ec-7f14-471f-b8c7-876382eb908f\",2592000]\";}' ] $_SESSION = [ '__flash' => [] '__returnUrl' => '/index.php' '__id' => 2 '__expire' => 1448379079 ] $_SERVER = [ 'HTTPS' => 'on' 'SSL_TLS_SNI' => 'humhub.domainname.de' 'HTTP_HOST' => 'humhub.domainname.de' 'HTTP_USER_AGENT' => 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10.10; rv:42.0) Gecko/20100101 Firefox/42.0' 'HTTP_ACCEPT' => 'application/json, text/javascript, */*; q=0.01' 'HTTP_ACCEPT_LANGUAGE' => 'de,en-US;q=0.7,en;q=0.3' 'HTTP_ACCEPT_ENCODING' => 'gzip, deflate' 'HTTP_DNT' => '1' 'HTTP_X_CSRF_TOKEN' => 'MDlGa2VKYW9eYSUcVXspKUV1PyEtIFUDVl80OhECBhdxCCkuVx1QJw==' 'HTTP_X_REQUESTED_WITH' => 'XMLHttpRequest' 'HTTP_REFERER' => 'https://humhub.domainname.de/index.php?r=admin%2Flogging' 'HTTP_COOKIE' => 'language=32e2115501b4a47845f68a6a07b73d99137bb6b8722808215f5a98e7f6acd5b7a%3A2%3A%7Bi%3A0%3Bs%3A8%3A%22language%22%3Bi%3A1%3Bs%3A2%3A%22de%22%3B%7D; mp_b5e6ddf58b2d02245a7a19005d1cec48_mixpanel=%7B%22distinct_id%22%3A%20%22150a9fc0508119-055c9ee5a67315-48516a-1fa400-150a9fc0509271%22%2C%22%24initial_referrer%22%3A%20%22https%3A%2F%2Fwww.domainname.de%3A8000%2F%22%2C%22%24initial_referring_domain%22%3A%20%22www.domainname.de%3A8000%22%7D; PHPSESSID=t72gum1ifnceui6akpbmbtae73; _csrf=2afd7ac664c5802e76463c4af56452b8f9b40e720494d18287866ee4d2435c74a%3A2%3A%7Bi%3A0%3Bs%3A5%3A%22_csrf%22%3Bi%3A1%3Bs%3A32%3A%22nXcw01HFuLyJHj4lffrQtHgxA1oE2W1H%22%3B%7D; pm_groups-statistics-panel=expanded; pm_new-spaces-panel=expanded; pm_getting-started-panel=expanded; pm_space-members-panel=expanded; pm_user-spaces-panel=expanded; pm_profile-following-panel=expanded; _identity=ef55eb012b22a134ac156aecbcf0b9e68f3a062d503cb7a206ac5ddc5f0d2bbda%3A2%3A%7Bi%3A0%3Bs%3A9%3A%22_identity%22%3Bi%3A1%3Bs%3A50%3A%22%5B2%2C%2200cf18ec-7f14-471f-b8c7-876382eb908f%22%2C2592000%5D%22%3B%7D' 'HTTP_CONNECTION' => 'keep-alive' 'PATH' => '/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin' 'SERVER_SIGNATURE' => '<address>Apache/2.4.7 (Ubuntu) Server at humhub.domainname.de Port 443</address> ' 'SERVER_SOFTWARE' => 'Apache/2.4.7 (Ubuntu)' 'SERVER_NAME' => 'humhub.domainname.de' 'SERVER_ADDR' => '192.168.0.101' 'SERVER_PORT' => '443' 'REMOTE_ADDR' => '217.91.96.230' 'DOCUMENT_ROOT' => '/var/www/html/humhub' 'REQUEST_SCHEME' => 'https' 'CONTEXT_PREFIX' => '' 'CONTEXT_DOCUMENT_ROOT' => '/var/www/html/humhub' 'SERVER_ADMIN' => '[no address given]' 'SCRIPT_FILENAME' => '/var/www/html/humhub/index.php' 'REMOTE_PORT' => '54082' 'GATEWAY_INTERFACE' => 'CGI/1.1' 'SERVER_PROTOCOL' => 'HTTP/1.1' 'REQUEST_METHOD' => 'GET' 'QUERY_STRING' => 'r=notification%2Flist%2Findex&from=0&ajax=1' 'REQUEST_URI' => '/index.php?r=notification%2Flist%2Findex&from=0&ajax=1' 'SCRIPT_NAME' => '/index.php' 'PHP_SELF' => '/index.php' 'REQUEST_TIME_FLOAT' => 1448377679.598 'REQUEST_TIME' => 1448377679 ] 
```

WebServer logs showed no errors.



The system was migrated from pre-Yii2.0 versions to the current 0.20 and recently to 0.20.1 via the humhub-updater.
The problem is not new to 0.20.1 it was there with 0.20 too, but it might have concluded from the migration.

**Comment by luke-** [comment]
maybe there are some notifications not correctly migrated between yii1 and yii2 versions.

You can try to empty the notification table or run the integrity checker:

cd protected
php yii integrity/run

**Comment by pelennart** [comment]
I did run the integrity checker and it asked me to delete (yes|no) a couple of notifications that came from the task module. Nevertheless it did not remove the error.

I truncated the notification table and it worked so I had a closer look:
There were two notifications that were responsible, both from class *humhub\modules\space\notifications\InviteDeclined* with *originator user ids* that do not exist (anymore?) in the users table.
